### DISCUSSION TITLE
Tailwindcss not compiling components or utilites, but base works

### DISCUSSION THREAD

**Discussion by ali-mozaffari** [tailwindlabs/tailwindcss#8113] [Help]
I follow up the tialwindcss.com/doc installation guide, but just basic styles work for me. I have checked all the things many times but couldn't find the problem.
Please help me to solve this issue.

**Comment by adamwathan** [comment]
Sounds like your `content` configuration isn't set up properly and that Tailwind isn't scanning your templates. What does your `tailwind.config.js` file look like?
